### PR TITLE
Add stripe position property for TabsControl

### PR DESCRIPTION
**Pull request by kurianov-pavel**
Add stripe position property into TabsControl
Add StripePosition enum

### CODE REVIEW

**Comment by AndreiMisiukevich** [comment]
Good contribution! Congratulations!

File: PanCardView/Controls/TabsControl.cs [hunk 1]
```diff
@@ -70,6 +71,11 @@ public class TabsControl : AbsoluteLayout
 
         public static readonly BindableProperty UseParentAsBindingContextProperty = BindableProperty.Create(nameof(UseParentAsBindingContext), typeof(bool), typeof(TabsControl), true);
 
+        public static readonly BindableProperty StripePositionProperty = BindableProperty.Create(nameof(StripePosition), typeof(StripePosition), typeof(TabsControl), Enums.StripePosition.Bottom, propertyChanged: (bindable, oldValue, newValue) =>
```
Comment (AndreiMisiukevich) [inline PanCardView/Controls/TabsControl.cs]:
```suggestion
        public static readonly BindableProperty StripePositionProperty = BindableProperty.Create(nameof(StripePosition), typeof(StripePosition), typeof(TabsControl), StripePosition.Bottom, propertyChanged: (bindable, oldValue, newValue) =>
```

File: PanCardView/Enums/StripePostion.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```
Comment (AndreiMisiukevich) [inline PanCardView/Enums/StripePostion.cs]: Please remove these three lines above

**Comment by AndreiMisiukevich** [comment]
Please amend those two places, then I'll merge this PR

File: PanCardView/Controls/TabsControl.cs (hunk 1 above)
Comment (kurianov-pavel) [reply PanCardView/Controls/TabsControl.cs]: Committed that change, thanks

**Comment by kurianov-pavel** [comment]
Updated

File: PanCardView/Enums/StripePostion.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace PanCardView.Enums
+{
+    public enum StripePosition
+    {
+        Top,
```
Comment (AndreiMisiukevich) [inline PanCardView/Enums/StripePostion.cs]: Bottom,
Top

Please change the order of elements (Default value is Bottom, so it should be the first element)

**Comment by kurianov-pavel** [comment]
Updated

**Pull request merged**